inference-engine/thirdparty/clDNN/tests/test_cases/resample_gpu_test.cpp

```diff
@@ -765,6 +764,254 @@ INSTANTIATE_TEST_CASE_P(smoke,
+        prim.align_corners = params.align_corners;
+        prim.pads_begin = params.pads_begin;
+        prim.pads_end = params.pads_end;
+        topo.add(prim);
+
+        auto build_opts = build_options();
+        build_opts.set_option(build_option::outputs({"resample"}));
+        build_opts.set_option(build_option::force_implementations({ {"resample", {params.in_format, "resample_ref"}} }));
+
+        auto net = network(eng, topo, build_opts);
+        net.set_input_data("in", in_mem);
+
+        auto result = net.execute();
+        auto output = result.at("resample").get_memory();
+
+        std::string kernel = "";
+        for (auto& info : net.get_primitives_info()) {
+            if (info.original_id == "resample")
+                kernel = info.kernel_id;
+        }
+
+        double ref_exec_time = 0;
+        for (auto& pri : net.get_executed_primitives()) {
+            if (pri.first == "resample") {
+                for (auto& pi : pri.second.get_profiling_info()) {
+                    if (pi.name == "executing") {
+                        ref_exec_time = std::chrono::duration_cast<std::chrono::microseconds>(pi.value->value()).count() / 1000.0;
+                    }
+                }
+            }
+        }
+
+        // Execute resample_opt
+        auto eng_opt = cldnn::engine(cfg);
+
+        cldnn::topology topo_opt;
+        topo_opt.add(input_layout("in", in_layout));
+        auto prim_opt = resample("resample_opt", "in", params.output_size, params.num_filter, params.operation_type);
+        prim_opt.align_corners = params.align_corners;
+        prim_opt.pads_begin = params.pads_begin;
+        prim_opt.pads_end = params.pads_end;
+        topo_opt.add(prim_opt);
+
+        auto build_opts_opt = build_options();
+        build_opts_opt.set_option(build_option::outputs({"resample_opt"}));
+        build_opts.set_option(build_option::force_implementations({ {"resample_opt", {params.in_format, "resample_opt"}} }));
+
+        auto net_opt = network(eng_opt, topo_opt, build_opts_opt);
+
+        // Use in_mem from ref network
+        net_opt.set_input_data("in", in_mem);
+
+        auto result_opt = net_opt.execute();
+        auto output_opt = result_opt.at("resample_opt").get_memory();
+
+        std::string kernel_opt = "";
+        for (auto& info : net_opt.get_primitives_info()) {
+            if (info.original_id == "resample_opt")
+                kernel_opt = info.kernel_id;
+        }
+
+        double opt_exec_time = 0;
+        for (auto& pri : net_opt.get_executed_primitives()) {
+            if (pri.first == "resample_opt") {
+                for (auto& pi : pri.second.get_profiling_info()) {
+                    if (pi.name == "executing") {
+                        opt_exec_time = std::chrono::duration_cast<std::chrono::microseconds>(pi.value->value()).count() / 1000.0;
+                    }
+                }
+            }
+        }
+
+        if (enable_profiling) {
```

Comment:
Do we still need profiling code?

Reply:
Removed.

Reply:
I still see some dead code for profiling (opt_exec_time, ref_exec_time calculation)